src/main/java/com/autosale/shop/repository/LogsRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.autosale.shop.repository;
+
+public interface LogsRepository {
+
+    void log(String message);
```

Comment:
method names in repositories should represent SQL operations. So you can name it as "create" or "insert"